Order.js

```diff
@@ -0,0 +1,67 @@
+import Hamburger from './Hamburger.js';
+import Drink from './Drink.js';
+import Salad from './Salad.js';
+
+class Order {
+  constructor(...order) {
+    this.order = order;
+    this.isPaid = false;
+  }
+
+  getOrder() {
+    return this.order;
+  }
+
+  calculatePrice() {
+    let endPrice = 0;
+    this.order.map(item => endPrice += item.calculatePrice());
+    return endPrice;
+  }
+
+  calculateCalories() {
+    let endCalories = 0;
+    this.order.map(item => endCalories += item.calculateCalories());
+    return endCalories;
+  }
+
+  payment() {
```

Comment:
I would rename this method like `doPayment` because it's usually a verb - what does the method do.